### DISCUSSION TITLE
blog/push_github_pages_to_another_reposigory

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#545] [Announcements]
# blog/push_github_pages_to_another_reposigory

Push Github Pages To Another Reposigory 从私有仓库部署项目到另一仓库 项目部分信息不适合公开, 只希望将部分文件通过vuepress的方式托管到GitHub Pages, 所以需要在自动部署的基础上设置, 详细参考: push-to-another-repository-docs

https://blog.maxiaoteng.xyz/blog/push_github_pages_to_another_reposigory.html

**Comment by shelton-xiaoteng-ma** [comment]
push
